### DISCUSSION TITLE
Topology view not showing

### DISCUSSION THREAD

**Discussion by Frank17489** [mattieserver/netbox-topology-views#519] [Q&A] [answered]
netbox: 4.0.5
netbox-topology-views: 4.0.0

plugin installed
- pip install netbox-topology-views
- python3 manage.py migrate netbox_topology_views with no errors
- python3 manage.py collectstatic --no-input  - ok
     -- Result: 0 static files copied to '/opt/netbox-4.0.5/netbox/static', 544 unmodified.

- systemctrl restart netbox.service netbox-rq.service

in netbox exists the menu Topology Views + sub menues
under Images only Circuit, Power Pane and Patch Panel has an icon
under Topology I see the tab **Network** and **Filter** but I don't see any network graphic

I see in the documentation under **use** a select-formular for Topology - I can't find this

under /opt/netbox I found only this:

../venv/lib/python3.11/site-packages/netbox_topology_views
../venv/lib/python3.11/site-packages/netbox_topology_views/templates/netbox_topology_views
../venv/lib/python3.11/site-packages/netbox_topology_views/static/netbox_topology_views
../venv/lib/python3.11/site-packages/netbox_topology_views-4.0.0.dist-info
../netbox/static/netbox_topology_views

It sounds like another threat from 2022. 
I try the advices from them, but it doesn't work

thx Frank

**Comment by dreng** [comment, marked as answer]
Hi Frank,

everything you described sounds like normal behavior.

1. Try to assign an icon of your choice. You can do that under Images by clicking any icon. A popup should open and show available icons. Select the one you want to assign and click Save.
2. Go to the Filters tab. Optionally select some filter criteria and options. Scroll down and click the button to submit the form. If nothing is displayed, check your filter criteria. Sometimes it is helpful to check the option "Show unconnected". Otherwise you will see connected icons only which sometimes equals no icons at all.

**Comment by Frank17489** [comment]
ah, ok, after some additional attempts I understand the principly now.  thx
